cognite/powerops/resync/models/base/asset_type.py

```diff
@@ -182,7 +182,8 @@ def _as_metadata(self) -> dict[str, str]:
             elif field.annotation in (str, int, float):
                 metadata[field_name] = str(value)
             elif field.annotation in (Optional[str], Optional[int], Optional[float]):
-                metadata[field_name] = str(value) if value is not None else ""
+                if value:
+                    metadata[field_name] = str(value)
```

Comment:
just ignore the field if there is no value which I guess makes sense if the field is Optional in the class definition